flytekit/control_plane/notifications.py

```diff
@@ -0,0 +1,101 @@
+from typing import List
+
+from flyteidl.admin import common_pb2 as _common_pb2
+
+from flytekit.models import common as _common_model
+from flytekit.models import execution as _execution_model
+from flytekit.models.core import execution as _core_execution_model
+
+Phases = List[_core_execution_model.WorkflowExecutionPhase]
+
+
+class Notification(_common_model.Notification):
```

Comment:
since notifications aren't really mutable and only really ever need to be read by the user and there aren't really any Admin api calls that deal with notifications, can we dispense with all these classes and just use the underlying models?